File: python/pangram/pangram.py
def is_pangram(sentence):
    alphabet = "abcdefghijklmnopqrstuvwxyz"
    letters = []
    for char in sentence.lower():
        if char not in letters and char in alphabet:
            letters.append(char)
    # i = 0
    # while i < len(sentence):
    #     if sentence[i].lower() not in letters and sentence[i].lower() in alphabet:
    #         letters.append(sentence[i].lower())
    #     i += 1
    return len(letters) == len(alphabet)

File: csharp/pangram/Pangram.cs
using System;
using System.Collections.Generic;

public static class Pangram
{
    static HashSet<char> alphabet = new HashSet<char>("abcdefghijklmnopqrstuvwxyz");

    public static bool IsPangram(string input)
    {
        return alphabet.IsSubsetOf(input.ToLower());
    }
}
